src/server/main_service.cc

```diff
@@ -72,6 +72,16 @@ ABSL_FLAG(bool, admin_nopass, false,
           "If set, would enable open admin access to console on the assigned port, without auth "
           "token needed.");
 
+ABSL_FLAG(string, replicaof, "",
+          "Empty by default. If not empty - specifies an IP and a port which "
+          "point to a running Dragonfly instance. The current Dragonfly "
+          "instance will replicate that machine. "
+          "Format should be <IPv4>:<PORT> or [<IPv6>]:<PORT>.");
+
+ABSL_FLAG(bool, continue_on_replication_fail, false,
+          "If set to true, failing to start replication using the "
+          "'--replicaof' flag would not exit Dragonfly. Disabled by default.");
```

Comment:
I also find this text slightly confusing.
Maybe:
"When true, Dragonfly will continue operation even when replication via `--replicaof` fails"
?

Reply:
phrasing has never been my strong suite :)